=== Content from bugs.debian.org_ff59ecbd_20250125_043019.html ===

# Debian Bug report logs - #854442 XXE vulnerability in python-openpyxl

[![version graph](version.cgi?package=python-openpyxl;found=openpyxl%2F2.3.0-2;fixed=openpyxl%2F2.3.0-3;height=2;width=2;absolute=0;collapse=1)](version.cgi?absolute=0;collapse=1;package=python-openpyxl;found=openpyxl%2F2.3.0-2;fixed=openpyxl%2F2.3.0-3;info=1)

Package:
[python-openpyxl](pkgreport.cgi?package=python-openpyxl);
Maintainer for [python-openpyxl](pkgreport.cgi?package=python-openpyxl) is [NeuroDebian Team <team@neuro.debian.net>](pkgreport.cgi?maint=team%40neuro.debian.net); Source for [python-openpyxl](pkgreport.cgi?package=python-openpyxl) is [src:openpyxl](pkgreport.cgi?src=openpyxl) ([PTS](https://tracker.debian.org/pkg/openpyxl), [buildd](https://buildd.debian.org/openpyxl), [popcon](https://qa.debian.org/popcon.php?package=openpyxl)).

Reported by: ["Ulikowski, Marcin" <marcin.ulikowski@f-secure.com>](pkgreport.cgi?submitter=marcin.ulikowski%40f-secure.com)

Date: Tue, 7 Feb 2017 09:36:02 UTC

Severity: important

Tags: security

Found in version openpyxl/2.3.0-2

Fixed in version openpyxl/2.3.0-3

**Done:** Yaroslav Halchenko <debian@onerussian.com>

Bug is archived. No further changes may be made.

Forwarded to <https://bitbucket.org/openpyxl/openpyxl/issues/749>

Display info messages

View this report as an [mbox folder](bugreport.cgi?bug=854442;mbox=yes), [status mbox](bugreport.cgi?bug=854442;mbox=yes;mboxstatus=yes), [maintainer mbox](bugreport.cgi?bug=854442;mbox=yes;mboxmaint=yes)

---

**Report forwarded**
to `debian-bugs-dist@lists.debian.org, NeuroDebian Team <team@neuro.debian.net>`:

`Bug#854442`; Package `python-openpyxl`.
(Tue, 07 Feb 2017 09:36:04 GMT) ([full text](bugreport.cgi?bug=854442;msg=2), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=2), [link](#1)).

---

**Acknowledgement sent**
to `"Ulikowski, Marcin" <marcin.ulikowski@f-secure.com>`:

New Bug report received and forwarded. Copy sent to `NeuroDebian Team <team@neuro.debian.net>`.

Your message had a Version: pseudo-header with an invalid package
version:

<= 2.3.5

please either use found or fixed to the control server with a correct
version, or reply to this report indicating the correct version so the
maintainer (or someone else) can correct it for you.

(Tue, 07 Feb 2017 09:36:04 GMT) ([full text](bugreport.cgi?bug=854442;msg=4), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=4), [link](#3)).

---

[Message #5](#5) received at submit@bugs.debian.org ([full text](bugreport.cgi?bug=854442;msg=5), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=5), reply):

![](/cgi-bin/libravatar.cgi?email=marcin.ulikowski%40f-secure.com)
From: "Ulikowski, Marcin" <marcin.ulikowski@f-secure.com>
To: "submit@bugs.debian.org" <submit@bugs.debian.org>
Subject: XXE vulnerability in python-openpyxl
Date: Tue, 7 Feb 2017 09:27:51 +0000
```
[[Message part 1](bugreport.cgi?att=0;bug=854442;msg=5) (text/plain, inline)]
```
```
Package: python-openpyxl
Version: <= 2.3.5

Openpyxl is vulnerable to XXE which allows reading local files and DoS an application. Openpyxl uses lxml which by default does resolve external entities and thus causes this vulnerability. This issue has been already reported to Openpyxl developers and was fixed about two weeks ago.

<https://bitbucket.org/openpyxl/openpyxl/commits/3b4905f428e1>

I'm attaching PoC code (xxe-poc.zip) which demonstrates this vulnerability. Script pyxl.py simulates a vulnerable application and takes Excel file as first argument. File blank_passwd.xlsx contains XXE payload which should read and store the contents of /etc/passwd.

--
Marcin Ulikowski
Senior Security Consultant | F-Secure CSS

```
```
[[xxe-poc.zip](bugreport.cgi?att=1;bug=854442;filename=xxe-poc.zip;msg=5) (application/x-zip-compressed, attachment)]
```

---

**Added tag(s) security.**
Request was from `Sebastien Delafond <seb@debian.org>`
to `control@bugs.debian.org`.
(Tue, 07 Feb 2017 09:45:04 GMT) ([full text](bugreport.cgi?bug=854442;msg=7), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=7), [link](#6)).

---

**Information forwarded**
to `debian-bugs-dist@lists.debian.org, NeuroDebian Team <team@neuro.debian.net>`:

`Bug#854442`; Package `python-openpyxl`.
(Tue, 07 Feb 2017 14:42:03 GMT) ([full text](bugreport.cgi?bug=854442;msg=9), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=9), [link](#8)).

---

**Acknowledgement sent**
to `Yaroslav Halchenko <debian@onerussian.com>`:

Extra info received and forwarded to list. Copy sent to `NeuroDebian Team <team@neuro.debian.net>`.
(Tue, 07 Feb 2017 14:42:03 GMT) ([full text](bugreport.cgi?bug=854442;msg=11), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=11), [link](#10)).

---

[Message #12](#12) received at submit@bugs.debian.org ([full text](bugreport.cgi?bug=854442;msg=12), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=12), reply):

![](/cgi-bin/libravatar.cgi?email=debian%40onerussian.com)
From: Yaroslav Halchenko <debian@onerussian.com>
To: "Ulikowski, Marcin" <marcin.ulikowski@f-secure.com>
Cc: "submit@bugs.debian.org" <submit@bugs.debian.org>
Subject: Re: Bug#854442: XXE vulnerability in python-openpyxl
Date: Tue, 7 Feb 2017 09:40:43 -0500
```
[[Message part 1](bugreport.cgi?att=0;bug=854442;msg=12) (text/plain, inline)]
```
```
thanks for the very detailed report!!! I have adopted that patch for our
now dated version of openpyxl  -- upload is coming shortly

On Tue, 07 Feb 2017, Ulikowski, Marcin wrote:

> Package: python-openpyxl
> Version: <= 2.3.5

> Openpyxl is vulnerable to XXE which allows reading local files and DoS an application. Openpyxl uses lxml which by default does resolve external entities and thus causes this vulnerability. This issue has been already reported to Openpyxl developers and was fixed about two weeks ago.

> <https://bitbucket.org/openpyxl/openpyxl/commits/3b4905f428e1>

> I'm attaching PoC code (xxe-poc.zip) which demonstrates this vulnerability. Script pyxl.py simulates a vulnerable application and takes Excel file as first argument. File blank_passwd.xlsx contains XXE payload which should read and store the contents of /etc/passwd.
--
Yaroslav O. Halchenko
Center for Open Neuroscience     <http://centerforopenneuroscience.org>
Dartmouth College, 419 Moore Hall, Hinman Box 6207, Hanover, NH 03755
Phone: +1 (603) 646-9834                       Fax: +1 (603) 646-1419
WWW:   <http://www.linkedin.com/in/yarik>

```
```
[[signature.asc](bugreport.cgi?att=1;bug=854442;filename=signature.asc;msg=12) (application/pgp-signature, inline)]
```

---

**Information forwarded**
to `debian-bugs-dist@lists.debian.org, NeuroDebian Team <team@neuro.debian.net>`:

`Bug#854442`; Package `python-openpyxl`.
(Tue, 07 Feb 2017 14:51:03 GMT) ([full text](bugreport.cgi?bug=854442;msg=14), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=14), [link](#13)).

---

**Acknowledgement sent**
to `Yaroslav Halchenko <debian@onerussian.com>`:

Extra info received and forwarded to list. Copy sent to `NeuroDebian Team <team@neuro.debian.net>`.
(Tue, 07 Feb 2017 14:51:03 GMT) ([full text](bugreport.cgi?bug=854442;msg=16), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=16), [link](#15)).

---

[Message #17](#17) received at 854442@bugs.debian.org ([full text](bugreport.cgi?bug=854442;msg=17), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=17), reply):

![](/cgi-bin/libravatar.cgi?email=debian%40onerussian.com)
From: Yaroslav Halchenko <debian@onerussian.com>
To: "Ulikowski, Marcin" <marcin.ulikowski@f-secure.com>
Cc: 854442@bugs.debian.org
Subject: Re: Bug#854442: XXE vulnerability in python-openpyxl
Date: Tue, 7 Feb 2017 09:48:48 -0500
```
[[Message part 1](bugreport.cgi?att=0;bug=854442;msg=17) (text/plain, inline)]
```
```

On Tue, 07 Feb 2017, Yaroslav Halchenko wrote:

> thanks for the very detailed report!!! I have adopted that patch for our
> now dated version of openpyxl  -- upload is coming shortly

unfortunately a blind adaptation of the patch wasn't sufficient, since
running your PoC code results in

$> python pyxl.py blank_passwd.xlsx
Traceback (most recent call last):
  File "pyxl.py", line 5, in <module>
    wb = load_workbook(filename = sys.argv[1])
  File "/usr/lib/python2.7/dist-packages/openpyxl/reader/excel.py", line 179, in load_workbook
    wb.properties = read_properties(archive.read(ARC_CORE))
  File "/usr/lib/python2.7/dist-packages/openpyxl/workbook/properties.py", line 122, in read_properties
    tag = localname(node)
  File "/usr/lib/python2.7/dist-packages/openpyxl/xml/functions.py", line 122, in localname
    m = NS_REGEX.match(node.tag)
TypeError: expected string or buffer

while most recent release 2.4.2 works correctly... before I dive into it -- any
clues what other changes should accompany the fix?  I am attaching my adapted
patch

--
Yaroslav O. Halchenko
Center for Open Neuroscience     <http://centerforopenneuroscience.org>
Dartmouth College, 419 Moore Hall, Hinman Box 6207, Hanover, NH 03755
Phone: +1 (603) 646-9834                       Fax: +1 (603) 646-1419
WWW:   <http://www.linkedin.com/in/yarik>

```
```
[[up_no_lxml](bugreport.cgi?att=1;bug=854442;filename=up_no_lxml;msg=17) (text/plain, attachment)]
```
```
[[signature.asc](bugreport.cgi?att=2;bug=854442;filename=signature.asc;msg=17) (application/pgp-signature, inline)]
```

---

**Information forwarded**
to `debian-bugs-dist@lists.debian.org, NeuroDebian Team <team@neuro.debian.net>`:

`Bug#854442`; Package `python-openpyxl`.
(Wed, 08 Feb 2017 22:33:05 GMT) ([full text](bugreport.cgi?bug=854442;msg=19), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=19), [link](#18)).

---

**Acknowledgement sent**
to `Markus Koschany <apo@debian.org>`:

Extra info received and forwarded to list. Copy sent to `NeuroDebian Team <team@neuro.debian.net>`.
(Wed, 08 Feb 2017 22:33:05 GMT) ([full text](bugreport.cgi?bug=854442;msg=21), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=21), [link](#20)).

---

[Message #22](#22) received at 854442@bugs.debian.org ([full text](bugreport.cgi?bug=854442;msg=22), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=22), reply):

![](/cgi-bin/libravatar.cgi?email=apo%40debian.org)
From: Markus Koschany <apo@debian.org>
To: marcin.ulikowski@f-secure.com, debian@onerussian.com
Cc: 854442@bugs.debian.org
Subject: Re: XXE vulnerability in python-openpyxl
Date: Wed, 8 Feb 2017 23:30:26 +0100
```
[[Message part 1](bugreport.cgi?att=0;bug=854442;msg=22) (text/plain, inline)]
```
```
Hi,

I am currently investigating if the versions of openpyxl in Wheezy and
Jessie are vulnerable. Apparently support for lxml was first introduced
in version 1.8. Wheezy and Jessie ship older versions though.

Is there another attack vector or can we assume that all versions
without lxml support are not affected?

Regards,

Markus

```
```
[[signature.asc](bugreport.cgi?att=1;bug=854442;filename=signature.asc;msg=22) (application/pgp-signature, attachment)]
```

---

**Set Bug forwarded-to-address to '<https://bitbucket.org/openpyxl/openpyxl/issues/749>'.**
Request was from `Salvatore Bonaccorso <carnil@debian.org>`
to `control@bugs.debian.org`.
(Wed, 15 Feb 2017 13:21:03 GMT) ([full text](bugreport.cgi?bug=854442;msg=24), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=24), [link](#23)).

---

**Reply sent**
to `Yaroslav Halchenko <debian@onerussian.com>`:

You have taken responsibility.
(Wed, 15 Feb 2017 14:54:03 GMT) ([full text](bugreport.cgi?bug=854442;msg=26), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=26), [link](#25)).

---

**Notification sent**
to `"Ulikowski, Marcin" <marcin.ulikowski@f-secure.com>`:

Bug acknowledged by developer.
(Wed, 15 Feb 2017 14:54:03 GMT) ([full text](bugreport.cgi?bug=854442;msg=28), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=28), [link](#27)).

---

[Message #29](#29) received at 854442-close@bugs.debian.org ([full text](bugreport.cgi?bug=854442;msg=29), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=29), reply):

![](/cgi-bin/libravatar.cgi?email=debian%40onerussian.com)
From: Yaroslav Halchenko <debian@onerussian.com>
To: 854442-close@bugs.debian.org
Subject: Bug#854442: fixed in openpyxl 2.3.0-3
Date: Wed, 15 Feb 2017 14:51:05 +0000
```
Source: openpyxl
Source-Version: 2.3.0-3

We believe that the bug you reported is fixed in the latest version of
openpyxl, which is due to be installed in the Debian FTP archive.

A summary of the changes between this version and the previous one is
attached.

Thank you for reporting the bug, which will now be closed.  If you
have further comments please address them to 854442@bugs.debian.org,
and the maintainer will reopen the bug report if appropriate.

Debian distribution maintenance software
pp.
Yaroslav Halchenko <debian@onerussian.com> (supplier of updated openpyxl package)

(This message was generated automatically at their request; if you
believe that there is a problem with it please contact the archive
administrators by mailing ftpmaster@ftp-master.debian.org)

-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Format: 1.8
Date: Tue, 07 Feb 2017 09:37:53 -0500
Source: openpyxl
Binary: python-openpyxl python3-openpyxl
Architecture: source all
Version: 2.3.0-3
Distribution: unstable
Urgency: medium
Maintainer: NeuroDebian Team <team@neuro.debian.net>
Changed-By: Yaroslav Halchenko <debian@onerussian.com>
Description:
 python-openpyxl - module to read/write OpenXML xlsx/xlsm files
 python3-openpyxl - module to read/write OpenXML xlsx/xlsm files
Closes: [854442](bugreport.cgi?bug=854442)
Changes:
 openpyxl (2.3.0-3) unstable; urgency=medium
 .
   * Do not resolve entities with lxml to avoid XXE vulnerability
     - patch up_no_lxml (Closes: #[854442](bugreport.cgi?bug=854442))
Checksums-Sha1:
 dc13b62a63628849d12cedb7a0bd40e2f6d5c9ab 2323 openpyxl_2.3.0-3.dsc
 b0e098d3b3350b1495d6cd5db134c3d7edf18ec2 5872 openpyxl_2.3.0-3.debian.tar.xz
 0161ac41c48b405fdd0f98ab172f18660a5ab159 6150 openpyxl_2.3.0-3_amd64.buildinfo
 ffa78477c12fa677f707bc28a75f6b29a0549e58 198944 python-openpyxl_2.3.0-3_all.deb
 7d7150f55b569d9b2fae8128546f63142d411098 198146 python3-openpyxl_2.3.0-3_all.deb
Checksums-Sha256:
 77427e84ffe07c6b2f436c961a78bcc759aa20c09d8738b38dc60ea9fe902409 2323 openpyxl_2.3.0-3.dsc
 c1d92e10479d1cf37cb0780f434d322ad800afb7ba98a5f918d0c3162be392b5 5872 openpyxl_2.3.0-3.debian.tar.xz
 c238d76cf14de36dfd09f89052c93cf63953162d7dfaa6cfad184aeba1a8655f 6150 openpyxl_2.3.0-3_amd64.buildinfo
 77885ecfdc15805aaa0e56d581956e10b5c64b38e96d66b7000ac256591ae2b4 198944 python-openpyxl_2.3.0-3_all.deb
 5b16a2ed151da38c70bcf32f04f9f265dff473c5711bb29485da533e034ee62c 198146 python3-openpyxl_2.3.0-3_all.deb
Files:
 f6ed85091c54389c0b2d9e1a083a0089 2323 python optional openpyxl_2.3.0-3.dsc
 570a100f7341ede0c5af604a4aa234ea 5872 python optional openpyxl_2.3.0-3.debian.tar.xz
 de327bba7b678f180b8bcebee74cd672 6150 python optional openpyxl_2.3.0-3_amd64.buildinfo
 7846059894566a1efd6e78fa95761d30 198944 python optional python-openpyxl_2.3.0-3_all.deb
 d45e53a0d6cc104ee1a08aa9f484bb7a 198146 python optional python3-openpyxl_2.3.0-3_all.deb

-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEExbkF8OjZ/ZZo/zZvot4jUGLaM/oFAlikX+UACgkQot4jUGLa
M/r4cQ/+K6KOQwoj9QwEcX4dn0Y/jDTdh3F/fesw9tLR+AscToJ2dTJXmvzCodTI
WubJHSuFFuLStzbHijK4248Y59Bj03Ae5f2koB2HkWFd5MH8efy9V4kBAFK0ibCj
FHBDXMW/sbVPRME8F9u8CmqxWtmPzP7fK4F6TVT9GG66x4p3yQzB/PS1RrTtj2MA
TdkdVeazmuZ2rJvl5MXrKFF1dPtyKCZD+fM4osO9XXHmHVGAemhQ1UlgmffZX14I
plbKifwX6Z8BDLzg6gOtEBcvc4vKjHi6fGMyIQbzLinZByB0LNolyn77zL45mVnA
gaov10d3TU/LCEfB7tgKADdTGzoitw36eocJz2uqNCTnG70+PG/tR1z07ege9LKZ
xHN5Q1StzylvflM6/L+yV5YlS9Xh74ztvo36LNMjYD/o9nI8u9Z7K43ACKL5nLqV
+EKAtcwNiOIK1gQE3OhJdcaypf79tCHwCz9qxIylUChnmbXChP591uDxHz4nuHuF
ESJH+kDJzrGIz39d5aAGSNFg99XOTWMvGW8pIW2qC7+sLug+KAD05PtbMDDb0lml
HE99laDaIp0J8MKOpPlsq/yYTeiGHlZY8k7a1YD4fkjMvDht7E7l5nz3EnjqSKRM
/MsLr5mfUCLG0Og3R3G6sUTQ9sTU2TMhFGRMqraXXPxmAsYwXrU=
=HUfI
-----END PGP SIGNATURE-----

```

---

**Marked as found in versions openpyxl/2.3.0-2.**
Request was from `Salvatore Bonaccorso <carnil@debian.org>`
to `control@bugs.debian.org`.
(Thu, 30 Mar 2017 18:06:03 GMT) ([full text](bugreport.cgi?bug=854442;msg=31), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=31), [link](#30)).

---

**Severity set to 'important' from 'normal'**
Request was from `Salvatore Bonaccorso <carnil@debian.org>`
to `control@bugs.debian.org`.
(Thu, 30 Mar 2017 18:06:04 GMT) ([full text](bugreport.cgi?bug=854442;msg=33), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=33), [link](#32)).

---

**Bug archived.**
Request was from `Debbugs Internal Request <owner@bugs.debian.org>`
to `internal_control@bugs.debian.org`.
(Fri, 28 Apr 2017 07:27:02 GMT) ([full text](bugreport.cgi?bug=854442;msg=35), [mbox](bugreport.cgi?bug=854442;mbox=yes;msg=35), [link](#34)).

---

Send a report that [this bug log contains spam](https://bugs.debian.org/cgi-bin/bugspam.cgi?bug=854442).

---

Debian bug tracking system administrator <owner@bugs.debian.org>.
Last modified:
Sat Jan 25 04:30:19 2025;
Machine Name:
buxtehude

[Debian Bug tracking system](https://www.debian.org/Bugs/)

Debbugs is free software and licensed under the terms of the GNU General
Public License version 2. The current version can be obtained
from <https://bugs.debian.org/debbugs-source/>.

Copyright © 1999 Darren O. Benham,
1997,2003 nCipher Corporation Ltd,
1994-97 Ian Jackson,
2005-2017 Don Armstrong, and many other contributors.



=== Content from bitbucket.org_65fe4b6c_20250125_043018.html ===

# Atlassian

JavaScript is disabled

You should enable JavaScript to work with this page.

# Atlassian

JavaScript load error

We tried to load scripts but something went wrong.

Please make sure that your network settings allow you to download scripts from the following domain:

**https://id-frontend.prod-east.frontend.public.atl-paas.net**



=== Content from www.openwall.com_13226ac1_20250125_043016.html ===


| [Openwall](/) * [Products](/)   + [Openwall GNU/\*/Linux   *server OS*](/Owl/)+ [Linux Kernel Runtime Guard](/lkrg/)+ [John the Ripper   *password cracker*](/john/)         - [Free & Open Source for any platform](/john/)- [in the cloud](/john/cloud/)- [Pro for Linux](/john/pro/linux/)- [Pro for macOS](/john/pro/macosx/)+ [Wordlists   *for password cracking*](/wordlists/)+ [passwdqc   *policy enforcement*](/passwdqc/)             - [Free & Open Source for Unix](/passwdqc/)- [Pro for Windows (Active Directory)](/passwdqc/windows/)+ [yescrypt   *KDF & password hashing*](/yescrypt/)+ [yespower   *Proof-of-Work (PoW)*](/yespower/)+ [crypt\_blowfish   *password hashing*](/crypt/)+ [phpass   *ditto in PHP*](/phpass/)+ [tcb   *better password shadowing*](/tcb/)+ [Pluggable Authentication Modules](/pam/)+ [scanlogd   *port scan detector*](/scanlogd/)+ [popa3d   *tiny POP3 daemon*](/popa3d/)+ [blists   *web interface to mailing lists*](/blists/)+ [msulogin   *single user mode login*](/msulogin/)+ [php\_mt\_seed   *mt\_rand() cracker*](/php_mt_seed/)* [Services](/services/)* Publications       + [Articles](/articles/)+ [Presentations](/presentations/)* Resources         + [Mailing lists](/lists/)+ [Community wiki](https://openwall.info/wiki/)+ [Source code repositories (GitHub)](https://github.com/openwall)+ [Source code repositories (CVSweb)](https://cvsweb.openwall.com)+ [File archive & mirrors](/mirrors/)+ [How to verify digital signatures](/signatures/)+ [OVE IDs](/ove/)* [What's new](/news) | |
| --- | --- |

| | [Follow @Openwall on Twitter for new release announcements and other news](https://twitter.com/openwall) | | --- | |
| --- | --- |

[[<prev]](4) [[next>]](6) [[thread-next>]](10) [[day]](.) [[month]](..) [[year]](../..) [[list]](../../..)
```

Message-ID: <20170207104854.331@usenet.piggo.com>
Date: Tue, 7 Feb 2017 09:57:39 +0000 (UTC)
From: Sébastien Delafond <seb@...ian.org>
To: oss-security@...ts.openwall.com
Subject: CVE request: XXE in Openpyxl

Hello,

the Debian Security Team would like to request a CVE for an XML XEE
discovered in Openpyxl by Marcin Ulikowski from F-Secure; Openpyxl
resolves external entities by default:

  <https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=854442>
  <https://bitbucket.org/openpyxl/openpyxl/commits/3b4905f428e1>

Cheers,

--Seb

```

[Powered by blists](https://www.openwall.com/blists/) - [more mailing lists](https://lists.openwall.net)

Please check out the
[Open Source Software Security Wiki](https://oss-security.openwall.org/wiki/), which is counterpart to this
[mailing list](https://oss-security.openwall.org/wiki/mailing-lists/oss-security).

Confused about [mailing lists](/lists/) and their use?
[Read about mailing lists on Wikipedia](https://en.wikipedia.org/wiki/Electronic_mailing_list)
and check out these
[guidelines on proper formatting of your messages](https://www.complang.tuwien.ac.at/anton/mail-news-errors.html).



=== Content from bitbucket.org_85a9d661_20250125_043017.html ===



